rdf_differ/entrypoints/ui/templates/dataset/view_dataset.html

```diff
@@ -40,8 +33,9 @@ <h1>You're viewing dataset: {{ dataset.dataset_id }}</h1>
         <tr>
             <th>Old version name</th>
             <th>{{ dataset.old_version_id }}</th>
-
         </tr>
         </tbody>
     </table>
+    <a href="{{ url_for('download_report', dataset_id=dataset.dataset_id) }}" id="report"
```

Comment:
I wonder how this magic works

Reply:
@costezki, by using the awesome power of `jinja` and `python` I can explain this:
- the `{{ }}` syntax allows to write `python` code inside of it, and it will be evaluated and "show" the output
- [url_for](https://flask.palletsprojects.com/en/1.1.x/api/#flask.url_for) is a `flask` utility method that enerates a URL to the given endpoint with the method provided. since `download_report` requires the `dataset_id` to run, I provide it by getting it from the `dataset` variable.

> if you want code magic, you should try writing `ruby`.